src/ApiService/IntegrationTests/EventsTests.cs

```diff
@@ -62,4 +63,52 @@ public EventsTestBase(ITestOutputHelper output, IStorage storage)
         var eventData = await sr.ReadToEndAsync(); // read to make sure the SAS URL works
         eventData.Should().Contain(ping.PingId.ToString());
     }
+
+    [Fact]
+    public async Async.Task UserInfoIsDeserialized() {
```

Comment:
If you undo the `_deserializingFromBlobOptions` change, this test fails with " reading UserInfo is not supported..." as expected.